### ISSUE TITLE
About attribute predictor and re-scoring analysis.

### ISSUE THREAD

**Issue by GreenLimeSia** [genforce/sefa#12]
Dear @ShenYujun  : 

Thanks for your excellent works. It is very useful for my research and I will cite your work in my study. However, I have some issues with re-scoring analysis since the attribute predictor is lost (The same issue on InterfaceGAN). Would you mind releasing your pre-trained attribute predictor's code and model? 

Thank you again.
Best wishes.

**Comment by ShenYujun** [comment]
Please refer to [this repo](https://github.com/NVlabs/stylegan2/blob/315ac5494112c94f3af5902d7c492b4dc471d112/metrics/linear_separability.py#L136).

**Comment by GreenLimeSia** [comment]
Many thanks for your prompt reply. After reading this repo, I still haven't found how to train a predictor with an attribute of the pose. As you noted in the paper, do you use the 5-point facial landmarks? If so, can you give me some advice on how to take advantage of these key points as pose?

**Comment by ShenYujun** [comment]
A very simple solution as an example: to get the pose (i.e., left v.s. right), you can use the x-coordinate of the nose point or the x-coordinate of the middle point between two eyes.

**Comment by GreenLimeSia** [comment]
Sure, I am very grateful for your guidance. I will train a classifier for it.

**Comment by di-mi-ta** [comment]
Hi @ShenYujun, 

Could you release code for rescoring-analysis metric in order to reproduce the results in the paper? 

Thanks!
